package.json

```diff
@@ -8,31 +8,36 @@
   "type": "module",
   "exports": {
     ".": {
-      "browser": "./dist/crypto.web.mjs",
-      "bun": "./dist/crypto.web.mjs",
-      "deno": "./dist/crypto.web.mjs",
-      "edge-light": "./dist/crypto.web.mjs",
-      "edge-routine": "./dist/crypto.web.mjs",
-      "lagon": "./dist/crypto.web.mjs",
-      "netlify": "./dist/crypto.web.mjs",
-      "react-native": "./dist/crypto.web.mjs",
-      "wintercg": "./dist/crypto.web.mjs",
-      "worker": "./dist/crypto.web.mjs",
-      "workerd": "./dist/crypto.web.mjs",
+      "browser": "./dist/crypto-web.mjs",
+      "bun": "./dist/crypto-web.mjs",
+      "deno": "./dist/crypto-web.mjs",
+      "edge-light": "./dist/crypto-web.mjs",
+      "edge-routine": "./dist/crypto-web.mjs",
+      "lagon": "./dist/crypto-web.mjs",
+      "netlify": "./dist/crypto-web.mjs",
+      "react-native": "./dist/crypto-web.mjs",
+      "wintercg": "./dist/crypto-web.mjs",
+      "worker": "./dist/crypto-web.mjs",
+      "workerd": "./dist/crypto-web.mjs",
       "node": {
-        "require": "./dist/crypto.node.cjs",
-        "import": "./dist/crypto.node.mjs",
-        "types": "./dist/crypto.node.d.ts"
+        "types": "./dist/crypto-node.d.ts",
```

Comment:
Why is `.` to `-` change required?

Reply:
Indeed, `unbuild` treated `.node` and `.web` as file extensions, regardless of the trailing `.ts` suffix.

I have also sorted the `types`, `import` and `require` fields to match [publint's recommendations](https://publint.dev/rules#exports_types_should_be_first).

Reply:
Okay leet's have it for now will investigate later.